docs/en/operations/settings/settings.md

```diff
@@ -1258,4 +1258,55 @@ Possible values:
 
 Default value: 16.
 
+## low_cardinality_max_dictionary_size {#low_cardinality_max_dictionary_size}
+
+Sets a maximum size in rows of a shared global dictionary for the [LowCardinality](../../sql-reference/data-types/lowcardinality.md) data type that can be written to a storage file system. This setting prevents problems with RAM in case of unlimited dictionary growth. All the data that was not dictionary encoded are written in not encoded way as ClickHouse do for non-`LowCardinality`-columns.
```

Comment:
* Afaik there are no shared global dictionaries in LowCardinality.
* problems -> issues
* The last sentence is hard to read.